### PR TITLE
Update geom-text.r

### PR DESCRIPTION
**Pull request by larspijnappel**
As proposed, here's a PR for the documentation improvement regarding the unsupported `check_overlap` argument for `geom_label()` (#4331).

Note: this request replaces the previous PR #4332 as it can't be completed due to loss of the previous fork.

### CODE REVIEW

**Comment by thomasp85** [comment]
Can I get you to run roxygen to also update the docs

**Comment by larspijnappel** [comment]
No problem, although that would be another first time for me as well :-)

So, in my local branch I performed `devtools::document()` and the `man\geom_text.Rd` has been updated successfully. 

Do I just `pr_push()` again to complete all of it?

**Comment by thomasp85** [comment]
Yes - every change can be added to the PR using `pr_push()`

File: NAMESPACE [hunk 1]
```diff
@@ -489,6 +489,7 @@ export(scale_color_fermenter)
 export(scale_color_gradient)
 export(scale_color_gradient2)
 export(scale_color_gradientn)
+export(scale_color_gray)
```
Comment (thomasp85) [inline NAMESPACE:492]: Can I get you to remove the two new exports in the NAMESPACE... It is not your fault, but another PR did not get oxygenised and this is now causing your PR to fail check. Removing these lets us handle it in a different PR

File: NAMESPACE [hunk 2]
```diff
@@ -536,6 +537,7 @@ export(scale_fill_fermenter)
 export(scale_fill_gradient)
 export(scale_fill_gradient2)
 export(scale_fill_gradientn)
+export(scale_fill_gray)
```
Comment (thomasp85) [inline NAMESPACE:540]: same as above

**Review by thomasp85 — CHANGES_REQUESTED** [review]

**Review by thomasp85 — APPROVED** [review]
Thanks!

**Pull request merged**